src/raised-button.jsx

```diff
@@ -35,7 +35,7 @@ function getStyles(props, state) {
 
   const amount = (primary || secondary) ? 0.4 : 0.08;
 
-  let backgroundColor = raisedButton.color;
+  let backgroundColor = props.backgroundColor || raisedButton.color;
```

Comment:
Can you deconstruct it at the top of the `render()` function. Should be good after that :+1: